dnf/cli/main.py

```diff
@@ -80,6 +80,7 @@ def main(args, conf_class=Conf, cli_class=Cli, option_parser_class=OptionParser)
         return 1
     except Exception as e:
```

Comment:
@mhatina Probably we should realy remove tris part, because dnf should never touch this kind of exception.